pixi_lock/action.yaml

```diff
@@ -30,10 +30,17 @@ runs:
         key: pixi|${{ env.TODAY }}|${{ hashFiles('pixi.toml') }}
     - uses: prefix-dev/setup-pixi@v0.8.1
       if: steps.cache.outputs.cache-hit != 'true'
-      id: install
       with:
         pixi-version: ${{ inputs.pixi-version }}
         manifest-path: ${{ inputs.pixi-manifest-path }}
+        run-install: false
```

Comment:
Could you describe in words what is the effect of this change? I'm not sure I follow.

Reply:
`run-install` will run `pixi install` solve and then set up the default environment. This has been okay so far, as there have been a low number of packages in the default environment. However, with the recent changes suggested in HoloViews, this no longer seems reasonable. So, I run a command below, which only does the lockstep. 

I could update the comment below:
``` yaml
    - name: Run update  # Using `pixi update` so we don't need to create and download the default environment.
```

Reply:
Yep why not :)